NameLayerMain/src/vg/civcraft/mc/namelayer/group/Group.java

```diff
@@ -317,23 +318,38 @@ public PlayerType getCurrentRank(UUID uuid) {
 	 * @param type- The PlayerType to add. If a preexisting PlayerType is found, 
 	 * it will be overwritten.
 	 */
-	public void addMember(UUID uuid, PlayerType type) {
+	
+	public void addMember(UUID uuid, PlayerType type){
+		addMember(uuid,type,true);
```

Comment:
No, you did not keep the prior calling method the same; if I call `addMember`, behavior previously was a synchronous, immediate call to the database. Now, the behavior is changed; `addMember` by default is now asynchronous, as stated in my issue report.

You corrected this in another portion of the PR but did not correct it here.